addons/im_livechat/static/src/embed/common/feedback_panel/transcript_sender.js

```diff
@@ -25,7 +25,7 @@ export class TranscriptSender extends Component {
         this.isValidEmail = isValidEmail;
         this.livechatService = useService("im_livechat.livechat");
         this.state = useState({
-            email: "",
+            email: this.livechatService.store.self_partner?.email,
```

Comment:
Should remove usage of `livechatService` in favor of directly using `store` so that it can be added to the common bundle.